standard/standard.lua

```diff
@@ -0,0 +1,34 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Standard
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+local OpponentLibrary = Lua.import('Module:OpponentLibraries')
+
+return {
+	Array = Lua.import('Module:Array'),
+	Class = Lua.import('Module:Class'),
+	Condition = Lua.import('Module:Condition'),
+	DateExt = Lua.import('Module:Date/Ext'),
+	Game = Lua.import('Module:Game'),
```

Comment:
not sure this is needed in a general import
like we don't see it being user extremly often